docs/run/sessions.mdx

```diff
@@ -33,6 +33,10 @@ The basic workflow for sessions is as follows:
 * The job must go through the normal queue to reactivate the session.
 </Admonition>
 
+<video title="A line of users is shown. The one in the current session becomes larger then the rest and has rotating arrows around them, indicating that they are the priority. Boxes (jobs) are seen being sent to the chandelier from the current user. Other users begin to pop up during the animation, and their jobs appear beneath them as they wait for the prioritized user to end their session. The prioritized user finishes and the lines moves ahead toward the system, so the next person's jobs can begin processing." class="max-w-auto h-auto" controls>
```

Comment:
nit - can we move the animation to above line 32? So it's not sandwiched between two `Note` boxes.